src/Text/Pandoc/Readers/Org.hs

```diff
@@ -446,11 +456,12 @@ followingResultsBlock =
                                      *> block)
 
 codeBlock :: BlockProperties -> OrgParser (F Blocks)
-codeBlock blkProp = do
+codeBlock (_, blockType) = do
```

Comment:
The reason I wasn't pattern matching the argument was that this gives a slightly better abstraction level.  Now, if the `BlockProperties` type changes, one would have to change this function as well.

I'm okay with this change, as it leads to slightly simpler code, which is good.  The slightly cleaner, but probably less readable way would be define a new function for changing the indentation level of a `BlockProperty`.